### PR TITLE
Add testing for the inspec scaffolding

### PR DESCRIPTION
**Pull request by TheLunaticScripter**
Signed-off-by: John Snow <thelunaticscripter@outlook.com>

Start adding testing for the `scaffolding-chef-inspec`.

### CODE REVIEW

File: scaffolding-chef-inspec/tests/user-linux-default/inspec.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: user-linux-default
+title: Effortless Audit with default plan settings
+maintainer: CCP Team
```
Comment (echohack) [inline scaffolding-chef-inspec/tests/user-linux-default/inspec.yml]: Please change to: `The Habitat Maintainers humans@habitat.sh`

File: scaffolding-chef-inspec/tests/user-linux-default/inspec.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+copyright: CCP Team
```
Comment (echohack) [inline scaffolding-chef-inspec/tests/user-linux-default/inspec.yml]: Please remove this line, as the copyright is handled in the root of the repo.

File: scaffolding-chef-inspec/tests/user-linux-default/inspec.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+copyright_email: you@example.com
```
Comment (echohack) [inline scaffolding-chef-inspec/tests/user-linux-default/inspec.yml]: Please remove

File: scaffolding-chef-inspec/tests/user-linux-default/inspec.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+license: Apache-2.0
+summary: Effortless Audit Linux Baseline
+version: 0.1.0
```
Comment (echohack) [inline scaffolding-chef-inspec/tests/user-linux-default/inspec.yml]: Please add `pkg_upstream_url="https://inspec.io"`

File: scaffolding-chef-inspec/tests/user-linux-default/inspec.yml (hunk 1 above)
Comment (TheLunaticScripter) [reply scaffolding-chef-inspec/tests/user-linux-default/inspec.yml]: I don't think that is the correct maintainer. This is not a project that is part of the habitat project.

File: scaffolding-chef-inspec/tests/user-linux-default/inspec.yml (hunk 4 above)
Comment (TheLunaticScripter) [reply scaffolding-chef-inspec/tests/user-linux-default/inspec.yml]: this is part of testing I think these changes are unnecessary as they don't affect the visibility of the tests.

File: scaffolding-chef-inspec/tests/user-linux-default/inspec.yml (hunk 1 above)
Comment (echohack) [reply scaffolding-chef-inspec/tests/user-linux-default/inspec.yml]: I see -- that's the maintainer we have for every other core plan and plan in the repo. What do you think the correct maintainer should be?

File: scaffolding-chef-inspec/tests/user-linux-default/inspec.yml (hunk 4 above)
Comment (echohack) [reply scaffolding-chef-inspec/tests/user-linux-default/inspec.yml]: Including the pkg_upstream_url is part of the required metadata for plans we use, but I understand if for a test you think it's unnecessary, so we can ignore this one. :)

File: scaffolding-chef-inspec/tests/user-linux-default/inspec.yml (hunk 1 above)
Comment (TheLunaticScripter) [reply scaffolding-chef-inspec/tests/user-linux-default/inspec.yml]: This is not a core plan this is a custom library to help customers use our tools easier to use and to reduce the need for other projects that do similar things from needing to maintain the same code.

**Pull request merged**